trillian/ctfe/handlers.go

```diff
@@ -444,7 +444,7 @@ func getSTHConsistency(ctx context.Context, c LogContext, w http.ResponseWriter,
 		rsp, err := c.rpcClient.GetConsistencyProof(ctx, &req)
 		glog.V(2).Infof("%s: GetSTHConsistency <= grpc.GetConsistencyProof err=%v", c.LogPrefix, err)
 		if err != nil {
-			return http.StatusInternalServerError, fmt.Errorf("backend GetConsistencyProof request failed: %v", err)
+			return http.StatusBadRequest, fmt.Errorf("backend GetConsistencyProof request failed: %v", err)
```

Comment:
This probably needs to inspect the underlying status with `grpc.Code(err)` to figure out whether it's a bad request or a borked server etc.